kudos.yaml

```diff
@@ -1984,5 +1984,18 @@
   tags: 
     - memes
   image: your_bounty_is_bad_and_you_should_fill_bad.svg
-
-
+- name: polyglot_programmer
+  description: For those who love languages, programming or otherwise!
```

Comment:
@jakerockland can you put your address in there as `to_address` 
and also add a param called `artist` that has your github username in it. 

these params arent fully baked into the site yet.. (you cant see them on the frontend), but eventually when we build artist pages theyll matter